automod/triggers.go

```diff
@@ -448,6 +468,12 @@ func (caps *AllCapsTrigger) UserSettings() []*SettingDef {
 			Min:     1,
 			Max:     100,
 		},
+		{
+			Name:    "Also match Zalgo and confusable characters",
```

Comment:
Make it simple for a average user? What does Zalgo and confusable characters here mean?